recipes/jinja2/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version="2.8" %}
+
+package:
+  name: jinja2
+  version: {{ version }}
+
+source:
+  fn: Jinja2-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/J/Jinja2/Jinja2-{{ version }}.tar.gz
+  md5: edb51693fe22c53cee5403775c71a99e
+
+build:
+  number:  0
+  script: python -m pip install --no-deps .
```

Comment:
most of the time, I'm copying existing recipes from either the internal repo or conda-recipes.  Going to open another file is a flow-interruptor for me.  Given, it's much, much better than going to look something up online.